centos_ci.sh

```diff
@@ -367,7 +367,7 @@ else
     perform_docs_publish "$REPO_OWNER" "$BRANCH" "minishift/docs/ondemand/$REPO_OWNER-$BRANCH";
   elif [[ "$JOB_NAME" = "minishift-release" ]]; then
     perform_release $RSYNC_PASSWORD;
-  elif [[ "$JOB_NAME" = "minishift-nightly-b2d" ]]; then
+  elif [[ "$JOB_NAME" = "minishift-pr" ]]; then
```

Comment:
This needs to changed back before merging right?

Reply:
Yes @LalatenduMohanty